<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicModeWindow</class>
 <widget class="QWidget" name="BasicModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢乐连连看——基本模式</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
background-color: white;
border: 1px solid black;
color: black;
border-radius: 5px;
}

QPushButton:hover {
background-color: #eee;
}

QPushButton:pressed {
background-color:#ccc;
}

QPushButton:disabled {
background-color: #ddd;
color: #aaa;
border-color: #aaa;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>20</y>
     <width>111</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="startGameBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pauseGameBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>暂停游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="hintBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>提示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rearrangeBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>重排</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timerLabel">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="buddy">
       <cstring></cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>520</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>640</width>
     <height>400</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
